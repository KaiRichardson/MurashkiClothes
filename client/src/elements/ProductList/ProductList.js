import styled from 'styled-components';

import { spacing } from 'utils';

export const ProductList = styled.section`
  display: flex;
  flex-wrap: wrap;
  padding: 0 ${spacing.lg};
`;
