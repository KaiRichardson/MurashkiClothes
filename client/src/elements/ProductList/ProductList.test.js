import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import { ProductList } from './ProductList';

describe('<ProductList /> tests', () => {
  it('matches the snapshot', () => {
    //* Act
    const { getByTestId } = render(<ProductList data-testid='testProductList' />);

    //* Assert
    const testProductList = getByTestId('testProductList');

    // Snapshot test
    expect(testProductList).toMatchSnapshot();
  });
});
