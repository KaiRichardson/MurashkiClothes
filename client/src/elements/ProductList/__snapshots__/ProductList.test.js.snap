// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductList /> tests matches the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  padding: 0 1.5rem;
}

<section
  class="c0"
  data-testid="testProductList"
/>
`;
